### ISSUE TITLE
DRAFT FE | Profile | Address Updates | Analytics 

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#46359]
> One thing to note is that we still have an accessibility violation in place for all of these use cases - there is an "Edit address" button under the user-entered address, which looks like a link instead of a button.
> 
> Addressing that is out of scope for this ticket, and will be part of the larger effort of finding a long term solution for address validation + address change messaging. I think it would be great to track how many people click those "Edit address" links that look like buttons so we can figure out whether or not people are actually using them.
> 
> Currently, they have the same GA event label as the main edit button, Profile - Edit Link - Mailing Address. I wonder if we can update the label so we can distinguish these links from the main edit button. That would be a FE task. Maybe we can call it Profile - Validation Edit Link - Mailing Address?

**Comment by andaleliz** [comment]
I believe you're right! @adamwhitlock1 would probably be a better person to confirm these links are gone, since he can look in the code. For further context, this is related to the "buttons that look like links" problem we had in the address validation flow if someone tried to save an address that can't be validated by the USPS.

**Comment by mtcA6** [comment]
@adamwhitlock1 could you check and confirm if we can close this out? I think this was resolved.

**Comment by mtcA6** [comment]
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/f3339015-63ad-4c48-a49e-a7658619bb2e)

**Comment by adamwhitlock1** [comment]
Yea this should be fine to close. I think this ticket is stale and has been resolved.
